### ISSUE TITLE
[Usability]A11y_Manage.IIS.Net_Web Server_Url Rewrite- Outbound Rules: Using keyboard, when focus comes to insert button user is unable to see the insert button on monitor visually and also the list is collapsing

### ISSUE THREAD

**Issue by InduPriya1805** [Microsoft/IIS.WebManager#183]
GitHubTags: #A11y_Manage.IIS.Net; #A11yUsable; #A11yTCS; #C+E-H; #DesktopWeb; #Win10; #.NETWebsites; #Edge; #Chrome; #Keyboard;

**Environment Details:**
URL: https://manage.iis.net/webserver/url-rewrite
Browser Details: Microsoft Edge 42.17134.1.0, Google Chrome 68.0.3440.106

**Repro Steps:**
1. Download and install "Microsoft IIS Administration" application.
2. Enable Internet information Services.
3. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4. Tab till Web Server and press enter
5. Tab till Url Rewrite and press enter
6. Tab till outbound rules tab and press enter
7. Tab till macros button and expand it. Verify Using keyboard, when focus comes to insert button user is unable to see the insert button on monitor visually and also the list is collapsing or not

**Actual Result:**
Using keyboard, when focus comes to insert button user is unable to see the insert button on monitor visually and also the list is collapsing

**Expected Result:**
Using keyboard, when focus comes to insert button user is unable to see the insert button on monitor visually and also the list should not collapse

**User Impact:**
If insert button is not visually appear on monitor users may get confused while working with the application

**Attachments:**
 
[Bug183_unable to see the insert button through keyboard tabbing.pptx](https://github.com/Microsoft/IIS.WebManager/files/2335412/Bug183_unable.to.see.the.insert.button.through.keyboard.tabbing.pptx)

**Comment by jhkimnew** [comment]
Fixed
